src/main/java/FindCommand.java

```diff
@@ -0,0 +1,24 @@
+import java.util.ArrayList;
+
+public class FindCommand extends Command {
+    private String description;
+
+    public FindCommand(String description) {
+        this.description = description;
+    }
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        System.out.println("Here are the matching tasks in your list:");
+        ArrayList<Task> matchingTasks = new ArrayList<>();
```

Comment:
I thought about this idea too! I chose to use streams in the end as the code is much simpler, this is a good idea!